src/components/Search.js

```diff
@@ -18,8 +19,28 @@ class Search extends React.Component {
         super(props);
         this.searchForm = React.createRef();
         this.searchInput = React.createRef();
+        this.state = {
+            autocompleteEvent: '',
```

Comment:
We might not need to track this event type. Search is next on the list for cleanup.